dynamic-theme/background.js

```diff
@@ -0,0 +1,50 @@
+var currentTheme = '';
+
+function setSun() {
+  if (currentTheme === 'sun') {
+    return;
+  }
+  currentTheme = 'sun';
+  browser.theme.update({
+    images: {
+      headerURL: 'sun.jpg',
+    },
+    colors: {
+      accentcolor: '#CF723F',
+      textcolor: '#111',
+    }
+  });
+}
+
+function setMoon() {
+  if (currentTheme === 'moon') {
+    return;
+  }
+  currentTheme = 'moon';
+  browser.theme.update({
+    images: {
+      headerURL: 'moon.jpg',
+    },
+    colors: {
+      accentcolor: '#000',
+      textcolor: '#fff',
+    }
+  });
+}
```

Comment:
I think I'd prefer code that declared const variables with the theme data in them, like:

    const day = {
      images: {
        headerURL: 'sun.jpg',
      },
      ...
    }

...then instead of `setSun()` and `setMoon()`, have `setTheme(theme)` called from `checkTime()` with the appropriate variable.

But this is just a matter of opinion.